tests/test_asset_test_sessions.py

```
@@ -0,0 +1,250 @@
+        row = _make_session_row()
+        table = mock_supabase_storage.supabase.table.return_value
+
+        # First query: not found
+        # Insert: fails with unique constraint
+        # Retry query: found
+        table.execute.side_effect = [
+            Mock(data=[]),  # SELECT: not found
+            Exception("duplicate key value violates unique constraint"),
+        ]
+        # The insert raises, so we mock it differently
+        original_execute = table.execute
```

Ruff (0.15.1)

[error] Local variable `original_execute` is assigned to but never used

Remove assignment to unused variable `original_execute`

(F841)